test/test_mail.py

```diff
@@ -34,6 +37,50 @@
 
 class UnitTests(unittest.TestCase):
 
+    def test_sendgridAPIKey(self):
+        """Tests if including SendGrid API will throw an Exception"""
+
+        #Minimum required to send an email
+        self.maxDiff = None
```

Comment:
PEP8 uses `snake_case`

```python
def test_sendgrid_api_key(self):
    self.max_diff = None

```
> [Function names should be lowercase, with words separated by underscores as necessary to improve readability.](http://pep8.org/#function-names)

> [Use the function naming rules: lowercase with words separated by underscores as necessary to improve readability.](http://pep8.org/#method-names-and-instance-variables)

Reply:
Good catch as well - I actually found `self.maxDiff` all over this test file before I modified it. I believe I fixed it all